<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transactions</class>
 <widget class="QDialog" name="transactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="LabelWel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelAccNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Account Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCurrBal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Balance: </string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelAmount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LELabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DepositButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WithdrawButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>200</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowHistoryButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HideHistoryButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AmountLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="HistoryLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
